app/controllers/steps/details/check_answers_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Steps::Details
+  class CheckAnswersController < Steps::DetailsStepController
+    def show
+      raise 'No tribunal case in session' unless current_tribunal_case
+
+      @fee_details_answers = FeeDetailsAnswersPresenter.new(current_tribunal_case)
+      @appeal_timeliness_answers = AppealTimelinessAnswersPresenter.new(current_tribunal_case)
+      @appellant_details = AppellantDetailsPresenter.new(current_tribunal_case)
+      @documents_submitted = DocumentsSubmittedPresenter.new(current_tribunal_case)
```

Comment:
We should probably go and rename some of these to fit in with how these concepts are named elsewhere in the app:

- `FeeDetailsAnswersPresenter`: So far we've used "cost" instead of "fee", but I think we should probably be using "fee" across the app because it makes more sense in the context of this app (there might be costs to the person appealing that aren't the fee)
- `AppealTimelinessAnswersPresenter`: We refer to "lateness" instead of "timeliness" elsewhere so should stick to that.
- `AppellantDetailsPresenter`: We call the appellant "taxpayer" throughout the app and should do that here too